mobile/lib/screens/kya/kya_lessons_page.dart

```diff
@@ -88,46 +101,128 @@ class _KyaLessonsPageState extends State<KyaLessonsPage> {
             ],
           ),
         ),
-        body: Container(
-          color: CustomColors.appBodyColor,
-          child: Column(
-            children: [
-              const Spacer(),
-              SizedBox(
-                height: 400,
-                child: ScrollablePositionedList.builder(
-                  padding: EdgeInsets.zero,
-                  scrollDirection: Axis.horizontal,
-                  itemCount: kya.lessons.length,
-                  physics: const NeverScrollableScrollPhysics(),
-                  itemBuilder: (context, index) {
-                    return Padding(
-                      padding: const EdgeInsets.only(
-                        left: 19,
-                        right: 19,
-                        bottom: 10,
-                      ),
-                      child: SizedBox(
-                        width: screenSize.width * 0.9,
-                        child: _kyaCard(kya.lessons[index], index),
+        backgroundColor: CustomColors.appBodyColor,
+        body: Column(
+          children: [
+            // const SizedBox(
```

Comment:
unless otherwise, let us minimise comments unless we are just temporarily disabling some functionality